tests/foreman/cli/test_report.py

```diff
@@ -22,6 +22,8 @@
 
 from robottelo.cli.base import CLIReturnCodeError
 
+pytestmark = pytest.mark.e2e
```

Comment:
Could you educate me on what this does?

Reply:
@Griffin-Sullivan Sure, it means all tests under this module will be marked as e2e

Reply:
Seen it before and was assuming that's what happens but never knew for sure. Thanks!